pkg/util/openshift/util.go

```diff
@@ -0,0 +1,103 @@
+package openshift
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/argoproj-labs/argocd-autopilot/pkg/git"
+	"github.com/argoproj-labs/argocd-autopilot/pkg/kube"
+	apstore "github.com/argoproj-labs/argocd-autopilot/pkg/store"
+	"github.com/codefresh-io/cli-v2/pkg/log"
+	"github.com/codefresh-io/cli-v2/pkg/store"
+	apu "github.com/codefresh-io/cli-v2/pkg/util/aputil"
+	kubeutil "github.com/codefresh-io/cli-v2/pkg/util/kube"
+	ocSecurityV1 "github.com/openshift/api/security/v1"
```

Comment:
consider removing this dependency and switch with yaml file that you render the values into, or with kustomize build